### PR TITLE
FIX: Too many item message and add possibility to change it

### PR DESCRIPTION
**Pull request by Jhobean**
When droping an item to the ground and there was too many item, the message was display on the character journal instead of ON the item. It's because the update was not done and your client did not see the item. By putting an Update before, it's fix.

THis part fix the issue like coruja did on #64.

I added the possibility to customize the message by script. For exemple, a french shard don't want to see "Too many items here!"

Don't forget approuve the new script pack message: https://github.com/cbnolok/SphereX-Scriptpack/pull/60

### CODE REVIEW

**Comment by Nirad** [comment]
Do you need to add this mesage to the other script pack as well to ?

**Comment by cbnolok** [comment]
As a suggestion, some comments with old code are left there for a reason, it's better to preserve it instead of wiping it

**Comment by Jhobean** [comment]
@Nirad i have pr old of 1 year on other script pack (56d)   I dont do PR there anymore

File: src/game/items/CItem.cpp [hunk 1]
```diff
@@ -1561,29 +1561,31 @@ bool CItem::MoveToCheck( const CPointMap & pt, CChar * pCharMover )
...
-			if ( iItemCount > g_Cfg.m_iMaxItemComplexity )
-			{
-				Speak("Too many items here!");
-				iDecayTime = 60 * MSECS_PER_SEC;		// force decay (even when REGION_FLAG_NODECAY is set)
-				break;
-			}
+		++iItemCount;
+		if ( iItemCount > g_Cfg.m_iMaxItemComplexity )
+		{
+			Speak(g_Cfg.GetDefaultMsg(DEFMSG_TOO_MANY_ITEM));
```
Comment (cbnolok) [inline src/game/items/CItem.cpp:1582]: Can you rename it to DEFMSG_TOO_MANY_ITEMS please?

**Review by cbnolok — CHANGES_REQUESTED** [review]

**Pull request merged**